Bottom: 4414dbc906d8ee5c1ef8b25deb7c5ddf8ad7bd29
Top:    a141061168183ec7a6c90a143c43dba9eae4dff9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 21:22:54 +0100

Refresh of rename-hidecrosshair-and-resto

---

diff --git a/src/crosshair.h b/src/crosshair.h
index ff16b1e..efa6679 100644
--- a/src/crosshair.h
+++ b/src/crosshair.h
@@ -46,6 +46,8 @@
 #define	STATE_SECOND	1
 #define	STATE_THIRD		2
 
+void notify_crosshair_change (bool changes_complete);
+void notify_mark_change (bool changes_complete);
 void CrosshairOn (void);
 void CrosshairOff (void);
 void HideCrosshair (void);
